pi_piper.gemspec

```diff
@@ -2,14 +2,14 @@
 
 Gem::Specification.new do |s|
   s.name = "pi_piper"
-  s.version = "2.0.beta.11"
+  s.version = "2.0.0"
```

Comment:
Agreed, I think the above is enough to make a `2.0.0`. We can release this as `1.9.9`.